### DISCUSSION TITLE
How to restore from Backup.

### DISCUSSION THREAD

**Discussion by ar-rahman-alamin** [crater-invoice-inc/crater#863] [Q&A]
Hi, 
I would like to thank the developers for creating such an easy-to-use and functional web app. As I have started using the app for the last few months I've found out it has a great feature to backup all the data. But I couldn't find any documentation/tutorial/option to restore from backup. Please share if anyone knows the procedure to restore from backup.

Thank you
Al

**Comment by captain-joni** [comment]
I guess you found out for yourself, but in case you are still wondering: I found this perfect Repo explaining every bit of restoring/migrating a Crater Installation:
https://github.com/albeanski/crater-invoice-backup-and-restore
